dali/test/python/auto_aug/test_auto_augment.py

```diff
@@ -210,8 +210,8 @@ def pipeline():
 @params(
     (False, "cpu", 256),
     (False, "gpu", 512),
-    (True, "cpu", 400),
-    (True, "gpu", 348),
+    (True, "cpu", 2000),
+    (True, "gpu", 2000),
```

Comment:
You don't plan to merge this fix in this PR, right?

Reply:
I do, why wouldn't I? This PR triggered the issue, so I fix it here. I think we agreed on this level of non-separation of concerns.

Reply:
Ah, I didn't get it that this PR is the only one where this issue happens. If so, of course go ahead and fix it here